contracts/optimism/L2ERC20ExtendedTokensBridge.sol

```diff
@@ -169,10 +169,15 @@ contract L2ERC20ExtendedTokensBridge is
         uint256 amount_
     ) internal returns (uint256) {
         if(l2Token_ == L2_TOKEN_REBASABLE) {
-            ERC20RebasableBridged(l2Token_).bridgeMintShares(to_, amount_);
-            return ERC20RebasableBridged(l2Token_).getTokensByShares(amount_);
+            IERC20Bridged(L2_TOKEN_NON_REBASABLE).bridgeMint(address(this), amount_);
```

Comment:
:thinking:  maybe we should allow calling `bridgeMint` and `bridgeBurn` only on the bridge contract address (not passing `from` or `to` args)

this way one would need to provide an explicit approval on `withdraw` for both stETH and wstETH

Reply:
We have decided to add a new method to the Rebasable token that allows the bridge to unwrap. It allowes user to withdraw without approval.